ansible/roles/debops.dovecot/defaults/main.yml

```diff
@@ -241,12 +277,12 @@ dovecot_pki_key: 'default.key'
 dovecot_ssl_required: True
 
                                                                    # ]]]
-# .. envvar:: dovecot_ssl_protocols [[[
+# .. envvar:: dovecot_ssl_min_protocol [[[
 #
 # SSL ciphers to use. On new distros you only specify the minimum ssl
 # protocol version Dovecot accepts, defaulting to TLSv1.
 # On old distros disable SSLv2,3 and allow TLSv1.0 or better.
-dovecot_ssl_protocols: '{{ "!SSLv2 !SSLv3"
+dovecot_ssl_min_protocol: '{{ "!SSLv2 !SSLv3"
```

Comment:
If you rename the variable also use the proper prefixing with two __ as you did for the new variables.

Reply:
Actually the new variables with two underlines (__) are actually wrong. As this role does not have the new variable name scheme with two underlines, I needed to stick to this "old format".